### DISCUSSION TITLE
How did you allow pip install to also install dependent packages

### DISCUSSION THREAD

**Discussion by ManglaAnjali** [eliasdabbas/advertools#212] [Q&A] [answered]
Hi Eliasdabbas,

I am new to python and used your advertools package to do an emoticon analysis. I must say it was super easy to use. Thank you.
Now I am trying to create my own package and it has internally dependency on other package. 
I did see when I was installing your package, it auto installed dependent package for me. 
How did you do that ?
It will be really helpful.
Thanks!

**Comment by eliasdabbas** [comment, marked as answer]
Thanks @ManglaAnjali 
Glad you liked it. 

If you are publishing a package on PYPI, then you can declare the dependencies in the [setup.py file](https://github.com/eliasdabbas/advertools/blob/93055877b0b2cc14ad59acce6833cf127c462818/setup.py#L51)

For more on packaging, you can checkout [this book](https://py-pkgs.org/)

**Reply by ManglaAnjali** [reply]
Awesome. Thanks @eliasdabbas :)
